src/test/kotlin/no/nav/syfo/cronjob/journalforing/DialogmoteVarselJournalforingCronjobSpek.kt

```diff
@@ -111,10 +112,13 @@ class DialogmoteVarselJournalforingCronjobSpek : Spek({
                 dokarkivBaseUrl = externalMockEnvironment.dokarkivMock.url,
             )
 
+            val cacheMock = mockk<RedisStore>()
```

Comment:
Mulig du kan bruke `(relaxed = true)` e.l. her for å slippe å mocke metodene under her.